### ISSUE TITLE
genericExpRename doesn't know about  mxExpectationMixture

### ISSUE THREAD

**Issue by tbates** [OpenMx/OpenMx#337]
via @mcneale It seems that the renaming helper in OpenMx doesn't know about mxExpectationMixture() expectations and so doesn't know how to rename them?

```R
oldFree = names(omxGetParameters(model, indep = TRUE, free = TRUE))
modelFixed = omxSetParameters(model, oldFree, free = FALSE, strict = TRUE, name = "_fixed")
```
> Error in genericExpRename(model@expectation, oldname, newname) : Not implemented yet

**Comment by RMKirkpatrick** [comment]
@tbates or @mcneale , could you provide more of a minimal reproducible example?

**Comment by tbates** [comment]
Here's a compact example from OpenMx

```R
require(OpenMx); data(myGrowthMixtureData)

class1 = mxModel("Class1", type="RAM", manifestVars=c("x1","x2","x3","x4","x5"), latentVars=c("intercept","slope"),
	mxPath(from=c("x1","x2","x3","x4","x5"), arrows=2, free=TRUE, values = c(1,1,1,1,1),labels=c("residual","residual","residual","residual","residual") ),
	mxPath(from=c("intercept","slope"), arrows=2, connect="unique.pairs", free=TRUE, values=c(1,.4,1), labels=c("vari1","cov1","vars1") ),
	mxPath(from="intercept", to=c("x1","x2","x3","x4","x5"), arrows=1, free=FALSE, values=c(1,1,1,1,1) ),
	mxPath(from="slope", to=c("x1","x2","x3","x4","x5"), arrows=1, free=FALSE, values=c(0,1,2,3,4) ),
	mxPath(from="one", to=c("x1","x2", "x3", "x4","x5"), arrows=1, free=FALSE, values=c(0,0,0,0,0) ),
	mxPath(from="one", to=c("intercept","slope"), arrows=1, free=TRUE,  values=c(0,-1), labels=c("meani1","means1") ),
	mxFitFunctionML(vector=TRUE)
)

class2 = mxModel(class1, name="Class2",
	mxPath(from=c("intercept","slope"), arrows=2, connect="unique.pairs", free=TRUE, values=c(1,.5,1), labels=c("vari2","cov2","vars2") ),
	mxPath(from="one", to=c("intercept", "slope"), arrows=1, free=TRUE, values=c(5,1), labels=c("meani2","means2") )
)

m1 = mxModel("Growth Mixture Model", mxData(myGrowthMixtureData, type="raw" ), class1, class2,
	mxMatrix(name="Props", type="Full", nrow=2, ncol=1, free=c(TRUE, FALSE), values=1, lbound=0.001, labels = c("p1","p2")),
	mxExpectationMixture(components=c('Class1', 'Class2'), weights='Props', scale='sum'),
	mxFitFunctionML()
)

old = names(omxGetParameters(m1, indep = TRUE, free = TRUE))
m2  = omxSetParameters(m1, old, free = FALSE, strict = TRUE, name = "_fixed")

```
> Error in genericExpRename(model@expectation, oldname, newname) : 
  Not implemented yet

**Comment by RMKirkpatrick** [comment]
Resolved with 3a67812.
